tests/testthat/test-summary_rows.R

```diff
@@ -635,6 +635,16 @@ test_that("Using `groups = FALSE` in `summary_rows()` returns data unchanged", {
   )
 })
 
+test_that("summary_rows() informs to use grand_summary_rows() if no groups are present (#1292).", {
+  expect_snapshot(
+    gt <- mtcars_short %>%
```

Comment:
```suggestion
  expect_snapshot(error = TRUE,
    gt <- mtcars_short %>%
```
if we decide to keep this an error.